code/controllers/subsystem/vote.dm

```diff
@@ -280,6 +281,7 @@ SUBSYSTEM_DEF(vote)
 			if(CONFIG_GET(flag/allow_vote_restart) || usr.client.holder)
 				if(min_restart_time < world.time)
 					initiate_vote("restart",usr.key)
+					CONFIG_SET(flag/allow_vote_restart, FALSE)
```

Comment:
I think its this line

Reply:
It's a feature.

Reply:
It looks like configs are saved between rounds, and I didn't catch it because on a private server you automatically have full privileges. Forcing voting to be enabled on server restart should fix this, and have the added benefit of making it less likely that an admin will log off leaving votes disabled by accident.